app/classifier/task.jsx

```diff
@@ -11,8 +11,9 @@ class Task extends React.Component {
 
   handleAnnotationChange(newAnnotation) {
     const { classification } = this.props;
-    classification.annotations[classification.annotations.length - 1] = newAnnotation;
-    classification.update('annotations');
+    const annotations = classification.annotations.slice();
+    annotations[annotations.length - 1] = newAnnotation;
```

Comment:
OOI why does this replace the last existing annotation with the new one, is it because the last one was a default? 

I tried to trace how the new default annotation gets it's actual value stored and it looks like it's in the specific task component on a change event. If that is so then any default will be updated to be a real one, right? Or i am way off track?

Reply:
This expects to get an updated annotation when a task changes. It simply overwrites your previous annotation, for this task, with the new one.

Reply:
The callback then forces a re-render by updating `state.annotations` in the classifier.

Reply:
right, got it.